.codecov.yml

```diff
@@ -1,3 +1,19 @@
+# Copyright 2025 Sonic Operations Ltd
+# This file is part of Sonic Client
```

Comment:
It might be too late, but "This file is part of the Sonic Client." might read better. Both versions are fine.